core/src/main/scala/com/spotify/featran/CrossingFeatureBuilder.scala

```diff
@@ -56,7 +56,7 @@ private case class Crossings(map: Crossings.MAP, keys: Set[String]) {
 }
 
 object CrossingFeatureBuilder {
-  def apply[F](fb: FeatureBuilder[F], crossings: Crossings): FeatureBuilder[F] =
+  @inline def apply[F](fb: FeatureBuilder[F], crossings: Crossings): FeatureBuilder[F] =
```

Comment:
This is for performance reasons? How noticeable is it?

Reply:
hard to say, i think it is the typical YMMV but i think it's good to hint the compiler to try to inline some of the things as much as possible to avoid extra calls.